R/filter_extreme.R

```diff
@@ -115,7 +115,7 @@ filter_extreme <- function(dataset,
   # group and sort input dataset
   tmp_obs_nr <- get_new_tmp_var(dataset)
   if (!is.null(by_vars)) {
-    assert_has_variables(dataset, vars2chr(by_vars))
+    assert_data_frame(dataset, required_vars = by_vars)
```

Comment:
I would move this to line 114. Then the `dataset` argument is also checked if `by_vars` is `NULL`.

Reply:
@bundfussr done!